README.md

```diff
@@ -74,14 +78,44 @@ class CronJob < ActiveJob::Base
 end
 ```
 
+### Example Job inheriting from CronJob
+
+Then, an example job that triggers E-Mail-sending with default cron time at
+noon every day:
+
+`app/jobs/noon_job.rb`:
+
+```ruby
+
+# Note that it inherits from `CronJob`
+class NoonJob < CronJob
+  # set the (default) cron expression
+  self.cron_expression '0 12 * * *'
```

Comment:
A `=` is missing.

Reply:
oh ya, fixed. Wanted to change CronJobs implementation, but didnt.